sotodlib/hwp/g3thwp.py

```diff
@@ -723,11 +720,13 @@ def write_solution_h5(self, tod, output=None, h5_address=None):
             data = self.load_data(start, end)
         except Exception as e:
             logger.error(f"Exception '{e}' thrown while loading HWP data. The specified encoder field is missing.")
+            self._write_solution_h5_logger = 'HWP data too short'
```

Comment:
This logger should be "Error in loading HWPdata"?

Reply:
This is not an error, just a short data length and no need to debug, so I prefer to leave it.